### DISCUSSION TITLE
Mobile App - Ability to Select Boards

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Ed/Community#44] [Enhancement request]
We have requirement of using multiple boards for the same org.

While referring to this https://github.com/Sunbird-Ed/Community/discussions/39
it seems there is hardcoding and assumption of one board only.

Is there any special reason its been set to use only one board per framework ?
if not can we enhance it to have multiple boards.

CC: @coolbung @santoshgandham

**Comment by swayangjit** [comment]
@baljeetsingh You can treat frameworks as board, create multi frameworks and attach them to the channel. This can solve your purpose.
To answer your question of "on board per framework". Initially, we had syllabus instead of board and one syllabus can have one board.
So that assumption carried forward till now.
cc: @sharathkashyap
